libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -40,6 +40,16 @@ import kotlinx.coroutines.channels.ReceiveChannel
 class MapboxNavigation(
     private val context: Context,
     private val accessToken: String?,
+    private val navigationOptions: NavigationOptions = NavigationOptions.Builder(
+        TWELVE_HOURS,
+        ROUNDING_INCREMENT_FIFTY,
+        distanceFormatter = MapboxDistanceFormatter(
+            context.applicationContext,
+            "ja",
+            METRIC,
+            ROUNDING_INCREMENT_FIFTY
```

Comment:
This is a relict of the sprint, could you look up what legacy version offered as default and use the same (unless some default values are worth changing)?

Reply:
It was the same value in legacy https://github.com/mapbox/mapbox-navigation-android/blob/master/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationOptions.kt#L85